lib/layout/dummyLayoutV1.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * dummylayout.js
+ *
+ * this dumps a fake graph
+ *
+ * @author John O'Brien.
+ */
+
+var Node = require('../model/node.js');
```

Comment:
Oh my. I never considered this, but it's probably a good idea for us to rename "node" to something else, so that we don't get node.js confused with Node.js. No action is required now. Just a thought.